src/Curators/Serializer/NewsArticleNormalizer.php

```diff
@@ -9,15 +9,31 @@
 
 final class NewsArticleNormalizer implements NormalizerInterface
 {
+    private $jsonLd = false;
```

Comment:
This property can get the `boolean` type.

Reply:
Right!